pkg/mcproducer/testdata/machineconfig-test.yaml

```diff
@@ -32,31 +37,15 @@ spec:
         - contents: |
             [Unit]
             Description=Pull oot kernel module image
-            ConditionPathExists=!/etc/systemd/system/on-prem-resolv-prepender.service
-            After=network-online.target
-            Requires=network-online.target
-            DefaultDependencies=no
-            [Service]
-            User=root
-            Type=oneshot
-            ExecStart=/usr/local/bin/pull-kernel-module-image.sh
-            PrivateTmp=yes
-            RemainAfterExit=no
-            TimeoutSec=900
-            [Install]
-            WantedBy=multi-user.target
-          enabled: true
-          name: "pull-kernel-module-image.service"
-        - contents: |
-            [Unit]
-            Description=Pull oot kernel module image in on-prem OCP cluster
-            ConditionPathExists=/etc/systemd/system/on-prem-resolv-prepender.service
             After=network-online.target
             After=NetworkManager-dispatcher.service
             DefaultDependencies=no
             [Service]
             User=root
             Type=oneshot
+            Environment="KERNEL_MODULE_IMAGE_FILEPATH=/var/lib/image_file_day1.tar"
+            Environment="WORKER_IMAGE=quay.io/edge-infrastructure/kernel-module-management-worker:latest"
+            Environment="KERNEL_MODULE_IMAGE=quay.io/project/repo:some-tag12"
```

Comment:
Nit: shall we keep the alphabetical order here?

Reply:
fixed